.github/workflows/main.yml

```diff
@@ -3,29 +3,45 @@ jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Install dependencies
-      run: |
-        sudo apt-get install -y rpm
-        sudo gem install fpm package_cloud
-    - name: Build
-      run: |
-        make deb
-        make rpm
-    - name: Deploy
-      if: ${{ github.ref == 'refs/heads/main' }}
-      run: |
-        PACKAGECLOUD_REPO="cs50/repo"
+      - uses: actions/checkout@v2.1.0
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Build packages for aarch64 platform
+        with:
+          arch: aarch64
+          distro: ubuntu20.04
+          dockerRunArgs: |
+            --volume ${PWD}/artifacts:/artifacts
```

Comment:
Is `artifacts/` a convention or should we use, e.g., `build/`?

Reply:
`artifacts` is the folder specified by GitHub action, so I think we might want to keep this naming convention.